packages/sync-actions/src/channels-actions.js

```diff
@@ -0,0 +1,38 @@
+import { buildBaseAttributesActions } from './utils/common-actions'
+import createBuildArrayActions, {
+  ADD_ACTIONS,
+  REMOVE_ACTIONS,
+} from './utils/create-build-array-actions'
+
+export const baseActionsList = [
+  { action: 'setKey', key: 'key' },
```

Comment:
this should be [`changeKey`](https://docs.commercetools.com/http-api-projects-channels.html#change-key)

it makes sense as well, since `key` is a required field on the draft